### PR TITLE
docs: Added Pandas-AI integration

### PR DESCRIPTION
**Pull request by Anush008**
[Preview](https://deploy-preview-745--condescending-goldwasser-91acf0.netlify.app/documentation/frameworks/pandas-ai/)

RELEASED https://github.com/Sinaptik-AI/pandas-ai/releases/tag/v2.0.23

### CODE REVIEW

**Comment by Anush008** [comment]
Waiting on https://github.com/Sinaptik-AI/pandas-ai/pull/1055.

File: qdrant-landing/content/documentation/frameworks/pandas-ai.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Pandas-AI
+weight: 2900
+---
+
+# Pandas-AI
+
+Pandas-AI is a Python library that uses a generative AI model to interpret natural language queries and translate them into Python code to interact with the data and return the results to the user.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/frameworks/pandas-ai.md:8]:
```suggestion
Pandas-AI is a Python library that uses a generative AI model to interpret natural language queries and translate them into Python code to interact with pandas data frames and return the final results to the user.
```

I was a bit confused if the names of `pandas-ai` and `pandas` are somehow related to each other.

File: qdrant-landing/content/documentation/frameworks/pandas-ai.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+os.environ["PANDASAI_API_KEY"] = "YOUR_API_KEY"
+
+agent = Agent(sales_by_country)
+agent.chat("Which are the top 5 countries by sales?")
+# OUTPUT: China, United States, Japan, Germany, Australia
+```
+
+## Qdrant support
+
+You can train Pandas-AI to understand your data better and improve result quality.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/frameworks/pandas-ai.md:55]:
```suggestion
You can train Pandas-AI to understand your data better and improve the quality of the results.
```

**Review by kacperlukawski — CHANGES_REQUESTED** [review]

**Review by kacperlukawski — APPROVED** [review]

**Comment by kacperlukawski** [comment]
I labeled the PR with "do not merge" label, as we're waiting for the PR to be merged in the main repo.

**Pull request merged**